.github/workflows/fullnode-execute-devnet-main.yaml

```diff
@@ -0,0 +1,51 @@
+# This workflow runs a public fullnode using the `main` branch,
+# connects the public fullnode to `devnet` and synchronizes the
+# node using execution syncing to verify that nothing has been broken.
+
+name: "fullnode-execute-devnet-main"
+on:
+  # Allow triggering manually
+  workflow_dispatch:
+    inputs:
+      GIT_SHA:
+        required: false
+        type: string
+        description: The git SHA1 to test. If not specified, Forge will check the latest commits on the current branch
```

Comment:
this input is not actually used -- can you remove it?

Reply:
Copy/pasta magic that I thought might be used. Removed from them all. 😄